### DISCUSSION TITLE
Zener in the V2.6 circuit

### DISCUSSION THREAD

**Discussion by EverlastEngineering** [bwack/C64-Saver-bwack#35] [General]
I modelled Z1 quickly and it doesn't act like I assumed it would. 

Here's how _I_ read the circuit's purpose:

When VIN goes to 8, the zener allows the difference to pass, so around 1.8V (1.2V?) goes to gate but that's not enough to shut down the mosfet. As VIN gets higher, so does the difference after the zener but the difference won't be enough to shut down the mosfet at any reasonable voltage.

@bwack was that the purpose of the zener, to "shut down" the circuit when it's _clearly_ higher than intended?

**Comment by bwack** [comment]
Its purpose is to protect the gate drain voltage from over and negative voltage.

**Comment by bwack** [comment]
It was a problem with saver V1 that total voltage over all the semiconductor junctions could cause breakdown. Thats why there is a resistor below to limit the surge current in a hotplug situation or transient. This usually happened when hotswapping or hotplugging where voltage differences can be very high.

**Reply by bwack** [reply]
Note that its purpose is not to shut off in these situations. It might as well turn on the mosfet, and pass some of the transient, but there is not enough energy in the transient raise the voltage in the c64 i belive I've tested this.

**Comment by dicemaN-Pete** [comment]
Hi @EverlastEngineering , I see you have fork of this project please can you detail the changes in v2.7, just wondering if I should build that or v2.6.  Also could you confirm the wattage of the zener, I can see from the BOM that it is 6.8v but not sure if I should use 1/2w 1w or something else. Thanks for helping.
